### PR TITLE
added quiz CRUD ui

### PR DESCRIPTION
**Pull request by lukecookssw**

### CODE REVIEW

File: src/UI/Pages/AddQuiz.razor [hunk 1]
```diff
@@ -47,7 +45,7 @@
         {
             exception.Redirect();
         }
-        catch (Exception ex)
+        catch (Exception)
```
Comment (matt-goldman) [inline src/UI/Pages/AddQuiz.razor:48]: Is this a debug try/catch?

File: src/UI/Pages/EditQuiz.razor [hunk 2]
```diff
@@ -56,7 +56,7 @@
         {
             exception.Redirect();
         }
-        catch (Exception ex)
+        catch (Exception)
```
Comment (matt-goldman) [inline src/UI/Pages/EditQuiz.razor:59]: Same question here

**Review by matt-goldman — APPROVED** [review]
Looks good, I have a question about some of the code you've been cleaning up here and wonder if we can go further.

File: src/UI/Pages/AddQuiz.razor (hunk 1 above)
Comment (lukecookssw) [reply src/UI/Pages/AddQuiz.razor]: Yep. Removed.

File: src/UI/Pages/EditQuiz.razor (hunk 2 above)
Comment (lukecookssw) [reply src/UI/Pages/EditQuiz.razor]: Removed.

**Pull request merged**